.github/workflows/ci.yml

```diff
@@ -215,6 +214,22 @@ jobs:
           # was created in this workflow.
           tls_ca_cert: ${{ env.TLS_CA_CERT }}
 
+      - name: Cleanup out dir
+        run: sudo rm -rf out_repo
+
+      - name: Run Bindplane Action (Directory)
+        uses: ./
+        with:
+          bindplane_remote_url: https://${{ env.MAIN_IP }}:3001
+          bindplane_username: admin
+          bindplane_password: admin
+          # This is a directory
+          processor_path: test/resources/processors
+          destination_path: test/resources/destinations/multi
```

Comment:
This test omits configs and sources, and inputs a processor and destination directory. Write back is disabled as we are not testing configs here.